src/routes/test/[slug]/api/current-position/+server.ts

```
@@ -0,0 +1,74 @@
+				headers: { 'Content-Type': 'application/json' },
+				body: JSON.stringify({ current_question_revision_id: question_revision_id })
+			}
+		);
+
+		if (!res.ok) {
+			let errorMessage = `Failed to save current position: ${res.status} ${res.statusText}`;
+
+			try {
+				const errorData = await res.json();
+				errorMessage = errorData?.detail || errorData?.message || errorData?.error || errorMessage;
+			} catch {}
```

ESLint

[error] Empty block statement.

(no-empty)